src/components/Icon/Icon.scss

```diff
@@ -27,192 +27,394 @@
 }
 
 .isColored {
-  color: color('white');
+  color: var(--p-text-on-interactive, color('white'));
```

Comment:
I'm unsure if this should always be white (the accent on twotone icons), or if it should be relative to the `surface` color. It looks a bit weird white in dark mode, which is the main difference.

Reply:
See screenshots in the description for what I'm talking about

Reply:
Seems like it should be `--p-surface` assuming the intent, in the current design system, is for the white to pop out against the sky background.